# Multiple detections with distance information
std_msgs/Header header
DetectionWithDistance[] detections
float32 inference_time_ms
float32 fusion_time_ms
int32 total_detections
int32 detections_with_distance  # How many detections got distance info